core-api/src/main/java/com/optimizely/ab/event/internal/EventFactory.java

```diff
@@ -168,6 +168,18 @@ public LogEvent createConversionEvent(@Nonnull ProjectConfig projectConfig,
         List<Attribute> attributesList = new ArrayList<Attribute>();
 
         for (Map.Entry<String, ?> entry : attributes.entrySet()) {
+            // Filter down to the types of values we're allowed to track.
+            // Don't allow Longs, BigIntegers, or BigDecimals - they /can/ theoretically be serialized as JSON numbers
+            // but may take on values that can't be faithfully parsed by the backend.
+            // https://github.com/optimizely/avro-schemas/blob/baeba19a7bd227aae67026474aec239110c5801d/event_batch/src/main/avro/EventBatch.avsc#L22
```

Comment:
This is a private repo. Please don't mention it here.

Reply:
Okay, fair enough. Can we link to https://developers.optimizely.com/x/events/api/#Attribute_ instead?